superchain/configs/sepolia/arena-z-testnet.toml

```diff
@@ -0,0 +1,64 @@
+name = "arena-z-testnet"
+public_rpc = "https://testnet-rpc.arena-z.gg"
+sequencer_rpc = "https://testnet-rpc.arena-z.gg"
+explorer = "https://testnet-explorer.arena-z.gg"
+superchain_level = 1
+governed_by_optimism = true
+superchain_time = 0
```

Comment:
Unless you want to inherit the standard hardfork times (defined [here](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/sepolia/superchain.toml#L5-L13)), you should delete this line (which effectively sets `superchain_time = nil`. 

If you decide you want to inherit hardforks at some point in the future, you can set this value to a unix timestamp and any hardfork whose standard activation is after the `superchain_time` standard will be pulled into this config .toml automatically (then repos like `op-geth` and `optimism` monorepo will need to update their import of superchain-registry chain config files to become aware of the new hardfork timestamp.